tests/serialize/data.js

```diff
@@ -99,7 +99,7 @@ var doNext = function (remaining) {
             return
           }
           doc = $rdf.sym($rdf.uri.join(right, base))
-          if (doc.uri.slice(0, 8) !== 'file:///') {
+          if (doc.uri.slice(0, 7) !== 'file://') {
```

Comment:
>  A file URI takes the form of
file://host/path
Note that when omitting host, the slash is not omitted (while "file:///foo.txt" is valid, "file://foo.txt" is not, although some interpreters manage to handle the latter).

So the node xhr behaviour is kind of incorrect.
It works, it's just ambiguous on what is loaded on the kb, as the nodes rapresenting the files are different on linux and windows which is not ideal. 

Windows will have
```
NamedNode {
  termType: 'Literal',
  value: 'file://C:\\destination\\folder/file.ttl' }
```
(process.cwd() is also using the reverse backslashes for windows which are then escaped by uri.sym I think. )
while in linux it will be 
```
NamedNode {
  termType: 'Literal',
  value: 'file:///D:/destination/folder/file.ttl' }
```
The slashes can be inverted at will since both node.fs and xmlhttprequest treat them as equal, but the initial slashes are still a problem.
This is a more general issue then the simple tests tho. Maybe it should be sanitized in the fetcher?